packages/legend-application/src/application/LegendApplicationConfig.ts

```diff
@@ -39,6 +39,7 @@ export interface LegendApplicationVersionData {
 export interface LegendApplicationConfigurationData {
   appName: string;
   env: string;
+  clientSideStorageKey: string;
```

Comment:
I think since the storage can be used with many different keys/indices, and we can make use of that to store many things, the best way is to do something like

```
storage: {
  settingsStoreKey: string;
}
```

Also, do you think this should be required?

Reply:
do you want to hardcode it instead of providing it in the config?

Reply:
yep, can be hardcoded for now, I don't see an use case yet where we would want to customize this.